plugins/SitesManager/API.php

```diff
@@ -1155,23 +1155,24 @@ public function setGlobalExcludedReferrers(string $excludedReferrers): void
 
         $excludedUrls = $this->checkAndReturnCommaSeparatedStringList($excludedReferrers);
 
-        foreach (explode(',', $excludedUrls) ?: [] as $url) {
-            // We allow urls to be provided:
-            // - fully qualified like http://example.url/path
-            // - without protocol like example.url/path
-            // - with subdomain wildcard like .example.url/path
-            $prefixedUrl = 'https://' . ltrim(preg_replace('/^https?:\/\//', '', $url), '.');
-            $parsedUrl = @parse_url($prefixedUrl);
-            if (false === $parsedUrl || !UrlHelper::isLookLikeUrl($prefixedUrl)) {
-                throw new Exception(Piwik::translate('SitesManager_ExceptionInvalidUrl', [$url]));
+            foreach (!empty($excludedUrls) ? explode(',', $excludedUrls) : [] as $url) {
+                // We allow urls to be provided:
+                // - fully qualified like http://example.url/path
+                // - without protocol like example.url/path
+                // - with subdomain wildcard like .example.url/path
+                $prefixedUrl = 'https://'.ltrim(preg_replace('/^https?:\/\//', '', $url), '.');
+                $parsedUrl = @parse_url($prefixedUrl);
+                if (false === $parsedUrl || filter_var($prefixedUrl,
+                        FILTER_VALIDATE_URL) === false || !UrlHelper::isLookLikeUrl($prefixedUrl)) {
```

Comment:
Are you aware that `FILTER_VALIDATE_URL` only allows ascii domains and might fail for internationalized domains?